scripts/hf_sim.py

```diff
@@ -203,8 +203,8 @@ def args_parser(cmd=None):
             # invalid arguments or -h
             comm.Abort()
 
-    if args.sim_bin is None:
-        args.sim_bin = binary_version.get_hf_binmod(args.version)
+        if args.sim_bin is None:
+            args.sim_bin = binary_version.get_hf_binmod(args.version)
```

Comment:
Are you merging with the block above / below too?

Reply:
Done